application/org.openjdk.jmc.console.ext.agent/plugin.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--   
+   Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2021 now.

Reply:
These files were created in 2020 but at the same time they'd be new files in JMC, would you prefer if we updated the headers to just 2021, or can we do 2020, 2021 like other files created in 2020?

Reply:
If they are new files, it's fine to change to just 2021 I think, but both variants are okay.